presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -414,8 +310,196 @@ as a part of SQL query by appending name with the table.
     ---------+------------------------------------------------------------------------------+-------------+--------------+--------------------+-----------------------------+--------------------------+----------------------+------------------+-------------------------------------------+--------------------------------------------+--------------+---------------+-------------
        0     | s3://my-bucket/ctas_nation/data/9f889274-6f74-4d28-8164-275eef99f660.parquet | PARQUET     |           25 |               1648 | {1=52, 2=222, 3=105, 4=757} | {1=25, 2=25, 3=25, 4=25} | {1=0, 2=0, 3=0, 4=0} |  NULL            | {1=0, 2=ALGERIA, 3=0, 4= haggle. careful} | {1=24, 2=VIETNAM, 3=4, 4=y final packaget} | NULL         | NULL          | NULL
 
+SQL Support
+-----------
+
+The Iceberg connector supports querying and manipulating Iceberg tables and schemas
+(databases). Here are the examples of SQL Operations supported from Presto :
+
+CREATE SCHEMA
+^^^^^^^^^^^^^^
+
+Create a new Iceberg schema named ``web`` that will store tables in an
+S3 bucket named ``my-bucket``::
+
+    CREATE SCHEMA iceberg.web
+    WITH (location = 's3://my-bucket/')
+
+CREATE TABLE
+^^^^^^^^^^^^^
+
+Create a new Iceberg table named ``page_views`` in the ``web`` schema
+that is stored using the ORC file format, partitioned by date and
+country::
+
+    CREATE TABLE iceberg.web.page_views (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioning = ARRAY['ds', 'country']
```

Comment:
Do we support partition column transformations? If yes, let's add a sub section here. If not, then add a line saying these are not supported yet.

Reply:
Presto Iceberg Connector currently already supports Bucket & Truncate transform with partition column in Iceberg Table. This feature request is to add transform support for Day, Month, Year, Hour
Created Issue for the same - https://github.com/prestodb/presto/issues/20570